components/infieldbutton/index.css

```
@@ -166,39 +124,35 @@
 		cursor: auto;
 	}
 
+	&:active {
+		transform: perspective(var(--spectrum-infield-button-downstate-perspective)) translateZ(var(--spectrum-component-size-difference-down));
+	}
+
 	&:focus-visible {
 		outline: none;
 	}
+}
 
-	/* Stacked in-field buttons */
-	/* Not currently in use (stepper uses Action Buttons) but adding the CSS so the option is there */
-	&.spectrum-InfieldButton--top,
-	&.spectrum-InfieldButton--bottom {
-		block-size: calc(var(--mod-infield-button-height, var(--spectrum-infield-button-height)) / 2);
-	}
+.spectrum-InfieldButton-inline {
+	display: flex;
 
-	&.spectrum-InfieldButton--top {
-		padding-block-end: var(--mod-infield-button-inner-edge-to-fill, var(--spectrum-infield-button-inner-edge-to-fill));
+	& .spectrum-InfieldButton {
+		--mod-infield-button-field-edge-to-icon: var(--spectrum-infield-button-inline-field-edge-to-icon);
+		inline-size: calc(var(--mod-infield-button-width, var(--spectrum-infield-button-width)) - var(--mod-infield-button-width, var(--spectrum-infield-button-edge-to-fill)));
```

@aramos-adobe I was just working on the number field branch, and I think this line probably needs to be modified. Right now, if someone sets a `--mod-infield-button-width`, this calc would subtract that from itself. 

```suggestion
inline-size: calc(var(--mod-infield-button-width, var(--spectrum-infield-button-width)) - var(--mod-infield-button-edge-to-fill, var(--spectrum-infield-button-edge-to-fill)));
```

This was probably meant to be a mod for the edge-to-fill.